From: Michael Hofmann <mh21@mh21.de>
Subject: Build with GTK+ 3.0

--- a/configure.ac
+++ b/configure.ac
@@ -21,7 +21,7 @@
 
 GLIB_GSETTINGS
 
-PKG_CHECK_MODULES(MULTILOADDEPS, [gtk+-2.0 cairo appindicator-0.1 glib-2.0 gio-unix-2.0 gmodule-2.0 libgtop-2.0])
+PKG_CHECK_MODULES(MULTILOADDEPS, [gtk+-3.0 cairo appindicator3-0.1 glib-2.0 gio-unix-2.0 gmodule-2.0 libgtop-2.0])
 
 AC_CONFIG_FILES([
 Makefile
--- a/debian/control
+++ b/debian/control
@@ -2,8 +2,8 @@
 Section: utils
 Priority: extra
 Maintainer: Michael Hofmann <mh21@mh21.de>
-Build-Depends: debhelper (>= 7.0.50~), libgtk2.0-dev, libcairo2-dev,
- libgtop2-dev, libappindicator-dev, valac, gnome-common, autopoint, dh-autoreconf
+Build-Depends: debhelper (>= 7.0.50~), libgtk3.0-dev, libcairo2-dev,
+ libgtop2-dev, libappindicator3-dev, valac, gnome-common, autopoint, dh-autoreconf
 Standards-Version: 3.9.1
 Homepage: https://launchpad.net/indicator-multiload
 Vcs-Bzr: http://code.launchpad.net/indicator-multiload
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -7,9 +7,9 @@
 AM_CFLAGS = -Wall -g
 
 VALAFLAGS =  \
-	--pkg=gtk+-2.0 \
+	--pkg=gtk+-3.0 \
 	--pkg=cairo \
-	--pkg=appindicator-0.1 \
+	--pkg=appindicator3-0.1 \
 	--pkg=glib-2.0 \
 	--pkg=gio-unix-2.0 \
 	--pkg=gmodule-2.0 \
